form/build.gradle.kts

```diff
@@ -53,6 +53,12 @@ android {
     composeOptions {
         kotlinCompilerExtensionVersion = libs.versions.kotlinCompilerExtensionVersion.get()
     }
+
+    testOptions {
+        unitTests {
+            isReturnDefaultValues = true
```

Comment:
what is this for?

Reply:
This was added because there was a error saying that the method "isEmpty()"in the Android's TextUtils class was not being mocked. Even though that method is not invoked anywhere in the code. The error happened when the method FormViewModel::processCalculatedItems()  called the FormCountingIdlingResource.increment() method.

Adding this configuration fixed this problem